### ISSUE TITLE
Uncaught TypeError: Cannot read property 'Constructor' of undefined

### ISSUE THREAD

**Issue by adamdelaney** [paypal/bootstrap-accessibility-plugin#114]
I'm getting`Uncaught TypeError: Cannot read property 'Constructor' of undefined` console.log errors when using the v1.0.4 in bootstrap-accessibility.min.js
